staging/src/k8s.io/apiserver/pkg/authentication/request/bearertoken/bearertoken.go

```diff
@@ -22,6 +22,11 @@ import (
 	"strings"
 
 	"k8s.io/apiserver/pkg/authentication/authenticator"
+	"k8s.io/apiserver/pkg/warning"
+)
+
+const (
+	invalidTokenWithSpaceWarning = "the provided Authorization header contains extra space before the bearer token, which is ignored"
```

Comment:
this is hard to tell whether the extra space is ignored, or the bearer token is ignored